src/markup/format/html.ts

```diff
@@ -82,6 +82,14 @@ function element(node: AbbreviationNode, index: number, items: AbbreviationNode[
             pushString(out, `</${name}>`);
             commentNodeAfter(node, state);
         }
+    } else if (config.options['jsx.enabled'] && !node.name) {
+        // Handle JSX fragment
```

Comment:
That defineitely should be tweaked, but Im not sure yet if its a valid way to handle jsx fragment in general

Reply:
No, nodes without name could be a `(group)`, e.g. `(ul>.item*4)+footer`